editor/modes/visual-editor/block.js

```diff
@@ -270,7 +313,81 @@ class VisualEditorBlock extends Component {
 
 	onKeyUp( event ) {
 		this.removeOrDeselect( event );
-		this.handleArrowKey( event );
+		//this.handleArrowKey( event );
+	}
+
+	handleToolbarTabCycle( event ) {
+		const { keyCode, shiftKey, target } = event;
```

Comment:
Yeah, the implementation is currently far from ideal, lots of duplicated code is the main thing irritating me at the moment. I mainly wanted to get some other eyes on it to see if people liked or hated the new keyboard nav.